sirepo/pkcli/admin.py

```diff
@@ -44,6 +50,25 @@ def audit_proprietary_lib_files(*uid):
 
 def create_examples():
     """Adds missing app examples to all users"""
+
+    def _create():
+        for sim_type in feature_config.cfg().sim_types:
+            names = [
```

Comment:
This is done twice, and really, three times. Create a function that returns `{name: example}` and then you can reuse in three places.